<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FirstQTClass</class>
 <widget class="QMainWindow" name="FirstQTClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FirstQT</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>501</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="workerStartButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::PreventContextMenu</enum>
    </property>
    <property name="text">
     <string>worker start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="workerEndButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>300</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>worker end</string>
    </property>
   </widget>
   <widget class="QPushButton" name="workerThreadStartButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>300</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>worker thread start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="workerThreadEndButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>300</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>worker thread end</string>
    </property>
   </widget>
   <widget class="QPushButton" name="memcpy_g_Button">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>300</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>memcpy_g</string>
    </property>
   </widget>
   <widget class="QPushButton" name="memcpyButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>330</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>memcpy</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>604</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpen">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuOption">
    <property name="title">
     <string>Option</string>
    </property>
    <addaction name="actionTest"/>
   </widget>
   <addaction name="menuOpen"/>
   <addaction name="menuOption"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="windowTitle">
    <string>Test</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionTest">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="firstqt.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>dockWidget</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>actionTest</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>39</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
